Website/Website/README.md

```diff
@@ -85,3 +85,10 @@ set DATAXDEV_INTERACTIVE_QUERY_LOCAL_SERVICE=<local URL of Interactive Query ser
 set DATAXDEV_SCHEMA_INFERENCE_LOCAL_SERVICE=<local URL of Schema Inference service>
 set DATAXDEV_LIVE_DATA_LOCAL_SERVICE=<local URL of Live Data service>
 ```
+
+### (Optional) How to target services hosted on AKS cluster
+If you are hosting DataX services on an AKS cluster, you can configure the website to target the services hosted on an AKS cluster by defining the following secret in the services keyvault. In the secret value: You only need to define the ones you are hosting on AKS cluster.
+The <External IP> should be added for each of the services where they can be listened to on the AKS cluster as specified in the example below:
+```
+{"Flow.InteractiveQueryService":"http://<External IP for Interactive Query Service>:5000","Flow.SchemaInferenceService":"http://<External IP for Schema Inference Service>:5000","Flow.ManagementService":"http://<External IP for Flow Management Service>:5000","Flow.LiveDataService":"http://<External IP for live Data Service>:5000"}
```

Comment:
let's add the secret (kubernetesServices) and the keyvault (this starts with kvServices) name here.

Reply:
Readme has been updated